test/src/unit-capi-consolidation.cc

```diff
@@ -4598,13 +4520,6 @@ TEST_CASE_METHOD(
     ConsolidationFx,
     "C API: Test consolidation, dense",
     "[capi][consolidation][dense]") {
```

Comment:
yes, so fragment consolidation and encryption are not supported for remote arrays. So for these ones I just removed the serialization wrappers because it's useless to test and didn't include them in rest CI. I have thought of introducing the `[non-rest]` tag for those tests to indicate that it just can't be ported, but missed to use it everywhere, hence your reasonable question.